index.html

```diff
@@ -179,6 +196,67 @@ <h3>
           }
         </pre>
       </section>
+      <section>
+        <h3>
+          Processing the `note_taking` member
+        </h3>
+        <p>
+          To <dfn>process the `note_taking` member</dfn>, given [=ordered map=]
+          |json:ordered map|, [=ordered map=] |manifest:ordered map|, [=URL=]
+          |manifest_URL:URL|, run the following during the
+          <a data-cite=
+          "appmanifest#dfn-processing-extension-point-of-web-manifest">extension
+          point</a> in [=processing a manifest=]:
+        </p>
+        <ol class="algorithm">
+          <li>If |json|["note_taking"] does not [=map/exist=], return.</li>
+          <li>If the type of |json|["note_taking"] is not [=ordered map=],
+            return.
+          </li>
+          <li>Set |manifest|["note_taking"] to a new [=ordered map=].</li>
+          <li>If |json|["note_taking"]["new_note_url"] does not [=map/exist=],
+            return.
+          </li>
+          <li>If the type of |json|["note_taking"]["new_note_url"] is not
```

Comment:
Aside: the next four lines appear so many times in the manifest spec, I think we should put it into a function "parse an in-scope URL", which parses a string as a URL relative to the manifest URL and returns failure if it isn't in scope.

Something for another time but just noting this down.

Reply:
Agreed.

Reply:
https://github.com/w3c/manifest/issues/986